fe/fe-core/src/main/java/com/starrocks/analysis/MVColumnOneChildPattern.java

```diff
@@ -42,6 +42,9 @@ public boolean match(Expr expr) {
         if (functionCallExpr.getChildren().size() != 1) {
             return false;
         }
+        if (functionCallExpr.getChild(0) instanceof CastExpr) {
+            return false;
+        }
         if (functionCallExpr.getChild(0).unwrapSlotRef() == null) {
```

Comment:
if FE upgrade to new version & user use cast create materialized view.
it will replay log OP_ALTER_JOB_V2 and run to CreateMaterializedViewStmt cause FE can not startup any more.

Reply:
https://github.com/StarRocks/starrocks/pull/78/commits/5a906371b4af4bc2cade49c0ad24e085c1f5d415